scripts/lint_site.sh

```diff
@@ -143,3 +143,5 @@ then
     echo "LINTING FAILED"
     exit 1
 fi
+
+ls -R
```

Comment:
You need to move this earlier, the script never gets here on failure.

Reply:
Thanks